client/app/src/services/network.service.js

```diff
@@ -268,10 +268,13 @@
       if (!network.forcepeer) {
         getFromPeer("/api/peers").then(function(response) {
           if (response.success) {
-            storageService.set("peers", response.peers.filter(function(peer) {
-              return peer.status == "OK";
-            }));
-            findGoodPeer(response.peers, 0);
+            getFromPeer('/api/peers/version').then(function(versionResponse) {
+              let peers = response.peers.filter(function(peer) {
```

Comment:
Does there need to be validation for response.success like the above api call here?

Reply:
Not a bad shout. It shouldn't be necessary but will add it anyway and will let it fallback on the stored peers if it fails 👍

Reply:
Pushed the change for you to take a look 👀